app/cypress/support/aws.config.ts

```diff
@@ -0,0 +1,11 @@
+import AWS from 'aws-sdk';
+
+// need to get from env vars
+AWS.config.update({
```

Comment:
This needs to be updated to pass in these values from environment variables. Need to clarify approach to follow existing implementations

Reply:
Is this to be left as is and updated in a future ticket?